android/app/src/main/kotlin/coop/polypoly/polypod/FeatureContainer.kt

```diff
@@ -115,6 +115,7 @@ class FeatureContainer(context: Context, attrs: AttributeSet? = null) :
 
     private fun loadFeature(feature: Feature) {
         webView.setBackgroundColor(feature.primaryColor)
+        Preferences.currentFeatureName = feature.fileName.dropLast(4)
```

Comment:
There's already `Feature.name`, which is essentially this (although the logic there is to cut out `.zip` from the string, rather than to cut the last four characters). I suppose we don't _need_ different logic here?

Reply:
Feature name is `Facebook Data Importer` in this case. All the spaces do not make it for a good folder path at all.

Reply:
Ah right, I forgot about the manifest name. Still not hot for the duplication - how about something like:

    val id: String get() = fileName.replace(".zip", "")
    val name: String get() = manifest.name ?: id